### PR TITLE
DSD-349:  Checkbox helper text

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes [DSD-349](https://jira.nypl.org/browse/DSD-349)

## **This PR does the following:**
- Adds `helperText` and `errorText` props to `Checkbox`

### Front End Review:
- [ ] View [the example in Storybook]()

### CODE REVIEW

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -81,6 +96,11 @@ const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
         {labelText && showLabel && (
           <Label htmlFor={checkboxID}>{labelText}</Label>
         )}
+        {(helperText || (errored && errorText)) && showLabel && (
```
Comment (EdwinGuzman) [inline src/components/Checkbox/Checkbox.tsx]: Similar comment to the other PR, I think you can replace `(helperText || (errored && errorText))` with `footnote`.

**Review by EdwinGuzman — COMMENTED** [review]
This looks good but can you also add two static examples for the error text and helper text? This is a similar comment to the Radio helper text PR #586

File: src/components/Checkbox/Checkbox.tsx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Checkbox/Checkbox.tsx]: Yes, this change worked well.

**Comment by bigfishdesign13** [comment]
@EdwinGuzman Changes made and ready for another review.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**